starlette/websockets.py

```diff
@@ -74,6 +74,9 @@ async def accept(
         subprotocol: str = None,
         headers: typing.Iterable[typing.Tuple[bytes, bytes]] = None,
     ) -> None:
+        if headers is None:
```

Comment:
As suggested by Kludex please remove the `headers` from message when it is None.